### PR TITLE
install /etc/crio/seccomp.json in rpm for ci

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@haircommander @umohnani8 @mrunalp

### CODE REVIEW

File: contrib/test/ci/seccomp.json [hunk 1]
```diff
@@ -0,0 +1,774 @@
+{
```
Comment (saschagrunert) [inline contrib/test/ci/seccomp.json]: Is this profile different from the default one? If yes, what are these? :innocent:

**Comment by umohnani8** [comment]
@lsm5 is this still needed?

**Pull request closed without merging**